### DISCUSSION TITLE
Component class modification

### DISCUSSION THREAD

**Discussion by beneyraheem** [rawilk/laravel-form-components#36] [Q&A]
Is there any easy way of customizing classes for the compnents, like 

`focus:border-blue-300 focus:ring-4 focus:ring-opacity-50 focus:ring-blue-400`

or do I need to overide and make new Custom Class?

**Comment by rawilk** [comment]
Converted to a discussion since it's more of a question.

Right now, the easiest way would probably be to override the component views. Depending on the component you may be able to target its class with css and apply your own colors there as well.

**Reply by beneyraheem** [reply]
Alright!. But I think it would be much better, if there is some ways to configure in the config.
